base/compiler/ssair/show.jl

```diff
@@ -2,15 +2,9 @@
 
 @nospecialize
 
-if Pair != Base.Pair
```

Comment:
This is here to load a separate, independent copy of the compiler. Admittedly that's not a particularly common workflow anymore, so maybe this is ok, but it was there for a reason.

Reply:
Interesting. Is that process documented somewhere?